Tests/Testo/Invoice/PaymentInformation/BitPayPaymentServiceTest.php

```
@@ -0,0 +1,256 @@
+{
+    /**
+     * @return LoggerInterface&m\MockInterface
+     */
+    private function makeLoggerInterfaceSpy(): LoggerInterface
+    {
+        /** @var LoggerInterface&m\MockInterface $mock */
+        $mock = m::spy(LoggerInterface::class);
+        return $mock;
+    }
+
+    private function makeSettingRepository(bool $sandbox = false): SettingRepository&m\MockInterface
```

PHPMD (2.15.0)

[error] The method makeSettingRepository has a boolean flag argument $sandbox, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)